examples/extending-image/postgresql-config/postgresql.conf

```diff
@@ -0,0 +1,528 @@
+# -----------------------------
+# PostgreSQL configuration file
+# -----------------------------
+#
```

Comment:
This example configuration file should be few-lines long only (this file is actually generated at postgresql build-time, and as such it would get staled very soon) and do something real..  and be included into the maint `postgresql.conf` file, or possibly transitively included into the configuration generated by our scripts...